### ISSUE TITLE
Implementation of Loss Eq(11)

### ISSUE THREAD

**Issue by TommyDzh** [Starlien95/GraphPrompt#8]
在pre_train.py 中通过reg_loss = compareloss(_pred,train_config["temperature"])计算 Eq(11)。可是_pred的shape是[batch_size, max_num_nodes, 3]，按我的理解_pred.shape不应该是[batch_size*max_num_nodes, hidden_dim, 3]吗。请问是不是我的理解有问题，谢谢！

**Comment by Starlien95** [comment]
我们更新了代码，解决了原有代码中的一些bug，包括您提到的问题
